Sources/Bot/API/Response.swift

```diff
@@ -30,7 +30,7 @@ private func linksInLinkHeader(_ header: String) -> [String: URL] {
 }
 
 /// A response from the GitHub API.
-internal struct Response: Hashable {
+public struct Response: Hashable {
```

Comment:
[nit] should we rename that to `GitHubResponse` or something to avoid ambiguity with `Vapor.Response`? (could be done in separate PR, just a nit)

Reply:
Nevertheless the comment this type is generic enough to be used in other environments but it's something we can do yes 👍

Reply:
Let's do it in a separate PR Olivier.